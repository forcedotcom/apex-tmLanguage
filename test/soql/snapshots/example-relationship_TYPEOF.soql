SELECT TYPEOF What WHEN Account THEN Phone, NumberOfEmployees WHEN Opportunity THEN Amount, CloseDate ELSE Name, Email END FROM Event
