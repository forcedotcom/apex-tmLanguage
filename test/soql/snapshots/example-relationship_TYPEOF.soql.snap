>SELECT TYPEOF What WHEN Account THEN Phone, NumberOfEmployees WHEN Opportunity THEN Amount, CloseDate ELSE Name, Email END FROM Event
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^^^^^ source.soql keyword.operator.query.apex
#             ^ source.soql
#              ^^^^ source.soql keyword.query.field.apex
#                  ^ source.soql
#                   ^^^^ source.soql keyword.query.field.apex
#                       ^ source.soql
#                        ^^^^^^^ source.soql keyword.query.field.apex
#                               ^ source.soql
#                                ^^^^ source.soql keyword.query.field.apex
#                                    ^ source.soql
#                                     ^^^^^ source.soql keyword.query.field.apex
#                                          ^ source.soql punctuation.separator.comma.apex
#                                           ^ source.soql
#                                            ^^^^^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                                             ^ source.soql
#                                                              ^^^^ source.soql keyword.query.field.apex
#                                                                  ^ source.soql
#                                                                   ^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                                                              ^ source.soql
#                                                                               ^^^^ source.soql keyword.query.field.apex
#                                                                                   ^ source.soql
#                                                                                    ^^^^^^ source.soql keyword.query.field.apex
#                                                                                          ^ source.soql punctuation.separator.comma.apex
#                                                                                           ^ source.soql
#                                                                                            ^^^^^^^^^ source.soql keyword.query.field.apex
#                                                                                                     ^ source.soql
#                                                                                                      ^^^^ source.soql keyword.query.field.apex
#                                                                                                          ^ source.soql
#                                                                                                           ^^^^ source.soql keyword.query.field.apex
#                                                                                                               ^ source.soql punctuation.separator.comma.apex
#                                                                                                                ^ source.soql
#                                                                                                                 ^^^^^ source.soql keyword.query.field.apex
#                                                                                                                      ^ source.soql
#                                                                                                                       ^^^ source.soql keyword.query.field.apex
#                                                                                                                          ^ source.soql
#                                                                                                                           ^^^^ source.soql keyword.operator.query.from.apex
#                                                                                                                               ^ source.soql
#                                                                                                                                ^^^^^ source.soql storage.type.apex
>